### ISSUE TITLE
Members Manager 2.0.0 - Profile Image Upload Error

### ISSUE THREAD

**Issue by joejvggithub** [vdm-io/Joomla-Members-Manager#1]
Steps to reproduce
Edit a Member
Click selecingt the file from your computer link
Select the file and click the Open button on Windows File Manager

Expected result
File is uploaded and thumbnail displays

Actual result
Alert message displays - There has been an error.
Staying on same page and uploading same image again produces no error alert message but no image is uploaded. If another image is displayed it displays same error message but second time does not and so on.

System information (as much as possible)
Windows: 7
MySql: 5.7.2.1
Apache Version: 2.4.33
PHP Version: 7.1.16
Joomla Version: 3.8.12
JCB Version:
Browser: Chrome 69.0.3497.100 (Official Build) (64-bit)

Additional comments
Great Component! - Thanks
![logo2](https://user-images.githubusercontent.com/602866/46080994-01252480-c161-11e8-82b0-2bbc5db9e056.jpg)

**Comment by Llewellynvdm** [comment]
I have fixed some issue in the member manger surrounding the Image uploader... I will push the changes soon. Then you can test and see if it was resolved.

**Comment by Llewellynvdm** [comment]
Remember you must set what UIKIT version member manger should use (in the component global options), if none is set, then the component my behave strange.

**Comment by joejvggithub** [comment]
Thanks Llewellyn. No hurry on fix. Your attention is greatly appreciated.

**Comment by Llewellynvdm** [comment]
Okay try this update out and let me know. [v2.0.1](https://github.com/vdm-io/Joomla-Members-Manager/releases/download/v2.0.1/JMM_v2.0.1.zip)

**Comment by joejvggithub** [comment]
Hi Llewellyn,

Status - Alert still displays in site area and admin.

**Comment by Llewellynvdm** [comment]
Can you give me a screenshot of this...

**Comment by Llewellynvdm** [comment]
IT all works perfectly on my side, and I have this running on many systems... so it seems strange.

**Comment by joejvggithub** [comment]
Hi Llewellyn,

I installed the latest version on a live server. Turns out error was only my local environment. Not sure if local environment is something you want to bother with. I'll look in my local logs and debug to see what's up. So it's documented I'm using:

WAMP 3.1.3
JCB v2.9.1
JMM_v2.0.1
Windows: 7
MySql: 5.7.2.1
Apache Version: 2.4.33
PHP Version: 7.1.16
Joomla Version: 3.8.12
Browser: Chrome 69.0.3497.100 (Official Build) (64-bit)

I'll get back as I have a good arsenal of debugging tools and use NetBeans v2.8.1 so I should be able to find something in short order. If not I'll let you know.

Regards,

Joe
